packages/airnode-node/src/adapters/http/worker.test.ts

```diff
@@ -17,30 +15,35 @@ describe('spawnNewApiCall', () => {
   };
   setLogOptions(logOptions);
 
+  const aggregatedApiCall = fixtures.buildAggregatedRegularApiCall();
+  const mockInvokeCommandInput = {
+    FunctionName: 'airnode-local02cce763-run',
+    Payload: new TextEncoder().encode(JSON.stringify({ aggregatedApiCall, logOptions, functionName: 'callApi' })),
+  };
+
+  beforeEach(() => {
+    mockLambdaClient.reset();
+  });
+
   it('handles remote AWS calls', async () => {
-    invokeMock.mockImplementationOnce((_params, callback) =>
-      callback(null, { Payload: JSON.stringify({ body: JSON.stringify({ ok: true, data: { value: '0x123' } }) }) })
-    );
-    const aggregatedApiCall = fixtures.buildAggregatedRegularApiCall();
+    mockLambdaClient.on(InvokeCommand, mockInvokeCommandInput).resolves({
+      Payload: new TextEncoder().encode(
```

Comment:
Nit: Create `encode` (or stringifyAndEncode) function which will stringify and encode, to make it less boilerplate...